engine/src/main/java/org/hibernate/validator/internal/engine/AbstractConfigurationImpl.java

```diff
@@ -208,6 +211,20 @@ public T parameterNameProvider(ParameterNameProvider parameterNameProvider) {
 		return thisAsT();
 	}
 
+	@Override
+	public T propertyNodeNameProvider(PropertyNodeNameProvider propertyNodeNameProvider) {
+		if ( LOG.isDebugEnabled() ) {
+			if ( propertyNodeNameProvider != null ) {
+				LOG.debug(
+						"Setting custom PropertyNodeNameProvider of type " + propertyNodeNameProvider.getClass()
```

Comment:
indentation

Reply:
It's how the project formatter indented it, so i don't want to argue :)